docs/content/cluster-scope/obtain_cluster_admin_access.md

```diff
@@ -0,0 +1,24 @@
+# Obtain Cluster Admin Priviledges in an Operate First Cluster
+
+## Pre-requisites
+
+1. Be a member of the sudoers group in the desired cluster
+2. Have access to the desired cluster via the CLI or the OpenShift
```

Comment:
What sort of access does one need to do impersonation from the console?

Reply:
To do impersonation from the console users need cluster-admin rights, unless they are a sudoer